pom.xml

```diff
@@ -308,6 +308,18 @@
             <version>1.10.19</version>
             <scope>test</scope>
         </dependency>
+        <!-- pin the dependency to fix CVE -->
+        <dependency>
```

Comment:
To change the version of a transitive dependency, its better to put that under a `depenednyManagement` section. Pinning a dependency will bring its own transitive dependencies, which can override and shadow other dependencies and their version. 

Also for the version lets put them under the properties section and reference them here.